machine/alphanetworks/alphanetworks_snh61a0_320f/machine.make

```diff
@@ -0,0 +1,47 @@
+# Alpha Networks SNH61A0-320F
+
+ONIE_ARCH ?= x86_64
+# Actually the ASIC vendor is Cavium here. However, there are no matching strings for now. 
+# That is why "unknown" is used for the vendor.
+SWITCH_ASIC_VENDOR = unknown
```

Comment:
Well, let's come up with a good string here and add it to the infrastructure.

The guideline is to use the stock ticker symbol, which in this case would be `cavm`.

The name is so short, however, that using `cavium` would also be fine.  Let's use `cavium`